src/output/dump_metrics.rs

```diff
@@ -311,3 +312,101 @@ fn dump_value(
     color!(stdout, White);
     writeln!(stdout, "{}", val)
 }
+
+pub fn dump_root_wmc(file_classes: &FileClasses) -> std::io::Result<()> {
```

Comment:
I don't see any caller for this function.

Reply:
I've the feeling that you're re-inventing the space stuff...
I'd prefer you integrate that stuff in `dump_root`.
If the idea was to avoid to print something for a non-obj language, then we should probably add something to configure the dumper and then dump the wmc stuff when it's required.